MetaMask/metamask-extension
# UX Multichain: Update settings title

This PR fixes the display of titles on the settings page.

Fixes #19458

## Screenshots/Screencaps

### Before

![Screenshot 2023-06-02 at 11 12 28 PM](https://github.com/MetaMask/metamask-extension/assets/39872794/0743c16b-9696-42d9-b42d-c16c9603c4da)


### After

![Screenshot 2023-06-02 at 11 30 57 PM](https://github.com/MetaMask/metamask-extension/assets/39872794/30109479-5a12-43b9-bd9d-dc9368a096ef)


![Screenshot 2023-06-02 at 11 30 31 PM](https://github.com/MetaMask/metamask-extension/assets/39872794/0e0384e5-d52a-4d56-aae0-c4f448985e28)

![Screenshot 2023-06-02 at 11 29 59 PM](https://github.com/MetaMask/metamask-extension/assets/39872794/34174c1d-c5c6-42f6-8f73-6e9f56a3899b)



## Manual Testing Steps

- Go to the Settings screen
- Check the title of the settings list with long titles


## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
